### ISSUE TITLE
Filter labels using an easy-to-interact-with results table

### ISSUE THREAD

**Issue by haesleinhuepf** [juglab/LabelEditor#17]
Hi @frauzufall, hi all,

there was an interesting online discussion on how to interact with label maps, focused on how to filter labels according to properties associated with labels, e.g. in an interactive results table. I know that such operations are technically feasible with the LabelEditor - it might just be that the user interface is missing yet. @VolkerH braught that idea up and I'm pretty sure he would be happy to support the project as early adopter and beta tester. I would be on board for that as well ;-)

https://forum.image.sc/t/simple-fiji-question-filter-particles-based-on-results-table-using-gui/39133

#nicetohave
#nohurry

Thanks for your awesome work! :-)

Cheers,
Robert

**Comment by VolkerH** [comment]
yes, I'd be happy to beta test. Wasn't aware of this LabelEditor repo until I saw this notification. Is this something that one can already play with (the README.md left me a bit clueless).

**Comment by haesleinhuepf** [comment]
@VolkerH 
https://forum.image.sc/t/bdv-labeleditor-objects-counter-ij2-for-dealing-with-labelings-in-imagej-usage/31635 ;-)
